README.md

```diff
@@ -8,8 +8,9 @@ The Google API client library for .NET enables access to Google APIs such as Dri
 Supported Platforms
 =================================
 
-* .NET Framework 4.5
-* Windows Store apps
+* .NET Framework 4.5 and 4.6
+* .NET Core (via netstandard1.3 support)
+* Windows 8 Apps
```

Comment:
That's odd - I thought I'd been doing a UWP app when I was investigating the Hololens bug. But if not, it's fine to just leave it off for the moment.